usagov/usagov-2021
# Remove webkit testing from benefits-finder-test.yml

## Jira Task 2108


https://cm-jira.usa.gov/browse/USAGOV-2108

## Description

This PR eliminates the webkit testing steps in the Benefit-finder integration testing pipelines. The testing has proven to be unreliable and we are removing it as unnecessary to avoid failures.

## Type of Changes

  - Add screenshot showing what it should look like
- [ x] Other

## Testing Instructions


Re-run pipelines after merge just to ensure they still run correctly which they should.

### Change Requirements

No new documentation required.


### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
